src/ebonite/client/base.py

```diff
@@ -300,3 +287,27 @@ def save_client_config(self, filepath):
         :param filepath: path to file
         """
         write(filepath, self, Ebonite)
+
+    def get_default_server(self):
+        """
+        :return: Default server implementation for this client
+        """
+        if self.default_server is None:
+            from ebonite.ext.flask import FlaskServer
+            self.default_server = FlaskServer()
+        return self.default_server
+
+    def get_default_environment(self):
```

Comment:
Previous implementation provided default (unfortunately it was named `localhost` because I messed with naming) environment for every environment type. Now we are back to hardcoding Docker environment (one of your main points against me in #55). Please describe the reasons behind this solution.

Reply:
Here are my thoughts
1) I understood that because we want it simple for local use (when we do no specify env explicitly), docker environment has to be hardcoded somewhere - so your solution from this point of view was (almost) ok
2) almost - because there was no way to change it - so I added class attribute and separate method to resolve the default env. It may be not very clean way, i admit, but at least it exists
3) as for deleted `.localhost_env` method - it only makes sense for docker, because what it localhost env for kubenetes, for example? even if we rename it to `.default_env`. Then I realized, actually we don't need default env for every runner, we only need one default runner (which is local docker). this rendered this method obsolete

Reply:
I do not fully agree with your points. But at least I understood motivation behind your solution. As far as I believe that we barely could ever face problems with these defaults (they are useful for testing and REPL and useless for prod) with any of implementations I resolve this comment.